crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/rpc.rs

```diff
@@ -94,13 +94,6 @@ impl CallFailure {
                     panic_data: err_data,
                 }
             }
-            EntryPointExecutionError::VirtualMachineExecutionErrorWithTrace { trace, .. } => {
```

Comment:
Why was this section changed?

Reply:
`VirtualMachineExecutionErrorWithTrace` has been removed